.travis.yml

```diff
@@ -4,6 +4,6 @@ node_js:
 services:
   - mongodb
 env:
-  - MONGODB_URI=mongodb://127.0.0.1/
+  - MONGODB_URI=mongodb://127.0.0.1/ JWT_SECRET=somesecret
```

Comment:
yeah this is fine, as long as it is not the same as the one we are using in production. (that one should be long and random)